k8s.gcr.io/ensure-prod-gclb.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/sh
+#
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script creates & configures the GCLB in front of the GCS bucket.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_PATH=$(readlink -f "$0")
+SCRIPT_DIR=$(dirname "$SCRIPT_PATH")
```

Comment:
You can also now delete line 23. Unnecessary use of readlink.
```suggestion
SCRIPT_DIR="$(dirname "${BASH_SOURCE[0]}")"
```

Reply:
Hmmm... copied this one from @thockin's scripts...

Reply:
This is the standard format we use for k/k hack/ scripts:
https://github.com/kubernetes/kubernetes/blob/bf22bbdeadae97748e0fe46bfd925aa14f3e3acf/hack/verify-gofmt.sh#L23-L24

Reply:
Works for me :-)   I'll change this PR and send a separate PR to fix other deviations.